Rustler wont link on OS X

Rustler wont build on OS X because of linking errors Output is below.

```
12:54:28 ➜  ~/dev ➜  git clone git@github.com:hansihe/rustler.git
Cloning into 'rustler'...
remote: Counting objects: 2542, done.
remote: Compressing objects: 100% (61/61), done.
remote: Total 2542 (delta 12), reused 36 (delta 9), pack-reused 2471
Receiving objects: 100% (2542/2542), 479.26 KiB | 2.90 MiB/s, done.
Resolving deltas: 100% (1649/1649), done.
12:55:54 ➜  ~/dev ➜  ll
total 216
drwxr-xr-x  26 rsimmons  staff   832B Jun 15 12:55 ./
drwxr-xr-x+ 73 rsimmons  staff   2.3K Jun 15 12:55 ../
-rw-r--r--@  1 rsimmons  staff    12K May 14 09:55 .DS_Store
-rwxr--r--@  1 rsimmons  staff   280B Dec 29 13:16 alter_kafka_topics.sh*
drwxr-xr-x@ 15 rsimmons  staff   480B Nov 16  2017 apache-activemq-5.15.2/
drwxr-xr-x  18 rsimmons  staff   576B May 30 01:43 book/
drwxr-xr-x  21 rsimmons  staff   672B Jun 15 10:05 bsp/
-rw-r--r--   1 rsimmons  staff    85K Apr 23 13:36 bsp.zip
drwxr-xr-x  10 rsimmons  staff   320B Jun 13 02:10 chimera/
drwxr-xr-x  15 rsimmons  staff   480B Jun 10 09:46 elixir-uuid/
drwxr-xr-x  16 rsimmons  staff   512B May 23 00:09 jason/
drwxr-xr-x   4 rsimmons  staff   128B Nov 10  2017 local_conf/
drwxr-xr-x  32 rsimmons  staff   1.0K May  6 00:30 otp/
drwxr-xr-x  15 rsimmons  staff   480B May 17 12:11 porcelain/
drwxr-xr-x  14 rsimmons  staff   448B Jun  5 23:13 rust-GSL/
drwxr-xr-x  11 rsimmons  staff   352B Jun  7 16:58 rust-demo/
drwxr-xr-x  14 rsimmons  staff   448B Jun 15 12:55 rustler/
drwxr-xr-x  10 rsimmons  staff   320B Jan 10 14:03 scrap/
drwxr-xr-x@ 15 rsimmons  staff   480B Nov 27  2017 star/
drwxr-xr-x  17 rsimmons  staff   544B Apr 10 14:03 stars-sports-platform/
drwxr-xr-x  15 rsimmons  staff   480B Apr 26  2017 starsdraft-test/
drwxr-xr-x  17 rsimmons  staff   544B Feb 17 08:11 step/
drwx------@ 17 rsimmons  staff   544B Jan 24 11:37 sumo-master@6dc6b2d3d45/
drwxr-xr-x  25 rsimmons  staff   800B May 31 07:25 victiv/
drwxr-xr-x  25 rsimmons  staff   800B Dec  2  2017 victiv_hotfix/
drwxr-xr-x  24 rsimmons  staff   768B Dec  2  2017 victiv_other/
12:55:57 ➜  ~/dev ➜  cd rustler
12:56:22 ➜  ~/dev/rustler ➜  git:(master) cargo build
    Updating registry `https://github.com/rust-lang/crates.io-index`
   Compiling void v1.0.2                                                        
   Compiling erlang_nif-sys v0.6.3
   Compiling unicode-xid v0.0.4
   Compiling quote v0.3.15
   Compiling unicode-segmentation v1.2.1
   Compiling lazy_static v1.0.1
   Compiling unreachable v0.1.1
   Compiling synom v0.11.3
   Compiling heck v0.3.0
   Compiling syn v0.11.11
   Compiling rustler v0.17.1-alpha.0 (file:///Users/rsimmons/dev/rustler/rustler)
warning: `#[must_use]` on methods is experimental (see issue #43302)
  --> rustler/src/types/binary.rs:47:5
   |
47 |     #[must_use]
   |     ^^^^^^^^^^^
   |
   = help: a nightly build of the compiler is required to enable this feature

warning: use of deprecated item 'std::ascii::AsciiExt': use inherent methods instead
 --> rustler/src/types/atom.rs:1:5
  |
1 | use std::ascii::AsciiExt;
  |     ^^^^^^^^^^^^^^^^^^^^
  |
  = note: #[warn(deprecated)] on by default

warning: unused import: `std::ascii::AsciiExt`
 --> rustler/src/types/atom.rs:1:5
  |
1 | use std::ascii::AsciiExt;
  |     ^^^^^^^^^^^^^^^^^^^^
  |
  = note: #[warn(unused_imports)] on by default

   Compiling rustler_codegen v0.17.0 (file:///Users/rsimmons/dev/rustler/rustler_codegen)
   Compiling rustler_test v0.1.0 (file:///Users/rsimmons/dev/rustler/rustler_tests)
warning: unused import: `NifTuple`
 --> rustler_tests/src/test_codegen.rs:3:10
  |
3 | #[derive(NifTuple)]
  |          ^^^^^^^^
  |
  = note: #[warn(unused_imports)] on by default

warning: unused import: `NifRecord`
  --> rustler_tests/src/test_codegen.rs:14:10
   |
14 | #[derive(NifRecord)]
   |          ^^^^^^^^^

error: linking with `cc` failed: exit code: 1
  |
  = note: "cc" "-m64" "-L" "/Users/rsimmons/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.133ztqntdc9j5bod.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.138pesuvab4dy035.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.13tprap6axy2fph1.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.16u6js6g0l3k1ic6.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.17erb31xn70viq4b.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.1ckp3bwk0jmrbky4.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.1cltj7mjlvtutgvw.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.1dqvxks6k2bzkxe.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.1fxq14vrn4hmbyv6.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.1im38lueib99jsk0.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.1jnlhyxj59jycbjv.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.1lm4ux8w9zr5umrf.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.1mvmz58owquyropc.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.1rv8n017ha9022uu.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.1vut2eft6nlujjxr.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.1woy8tiqimxj872l.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.1y16o1qfye96o7m0.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.1ygfkr7qu0ddj4qa.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.1zeawhkbeobww1zn.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.1zwd8n7bcl3vhvvh.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.236iz806sghm3wx.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.23tqyymcb18u96mb.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.25119y52vmwdiop0.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.25gju3vv0nlzzyym.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.285ux0t7vjsmmzc2.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.2at9w9lzqm60tjnr.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.2dh9yuzgscovothb.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.2gb2i6picgndn8mm.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.2gjib7n2y6fjv5hn.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.2h6js7r1t4ms8cda.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.2jqywn86b2gsqohu.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.2lyh15q6cjwzy18c.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.2pr8dy9r9x4e5q0l.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.2ufmh3pojqsi80dv.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.2ytek0d5g37lv8cb.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.2z7ffi9qqeig53uz.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.30wkhbrq68lviwyl.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.3171x0bwu82dptu7.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.36emqseb3shxwf8h.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.37alll1ouuz8aq1k.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.38ps4pa181wsnsy9.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.3ayaeypdcro9d6yk.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.3cx7oljifvb206q7.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.3f6klegmryfg2ey3.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.3f8rvg1lnyerot41.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.3hfnnvlph5ozv32c.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.3ik0x0hz6l66cx38.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.3oqznv5oym7l3m4e.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.3sh469pny04dzdkx.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.3vs55mhif3snkoiw.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.3vxgk64o4l297fkh.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.3z4myxuzecx47mt.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.436dotimmrgzkwfa.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.43v6g0y2xsxoggnt.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.45nf4z58qqykpcpi.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.45pc7c65foh9i35f.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.48721dc4k5qxei0u.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.49a7n47po4ttqjl7.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.49lx1q7cxvpykyv0.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.49qmpuk8w1opmi5q.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.49r6w5m6d8ypo1nz.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.4a6dgwgsyclvnmy1.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.4bwka44cn8t3u4h3.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.4cxyqmsl2ros1gyv.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.4dtnki5bgt8bvvrg.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.4ezmh1vbs95c5ack.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.4h3ia37ywo92ez09.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.4iqbs73z7t48pd85.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.4out7xipry4eqhpn.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.4sw9hz7i7fiueihz.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.4vlu23gxndyeguh8.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.4vp4wqj2v29i7mgy.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.4xq48u46a1pwiqn7.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.4yh8x2b62dcih00t.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.4ypvbwho0bu5tnww.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.4zcotb17g86jzx0v.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.51s1w397y42gpez1.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.52nuqctvqtgass6a.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.56dly8q07ws8ucdq.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.572bgozz9kcwe4zq.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.57k06xfugllsc526.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.5b1vh7iwaxrgdt9b.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.5by9xggibmun5kn9.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.5gf6du7k58s78kob.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.7p53qlves2rwxx0.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.8xzrsc1ux72v29j.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.98g0d9x8aw3akpe.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.9elsx31vb4it187.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.c6lbtaiefvx3wya.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.d6p2df93qm1h3gm.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.dyfjd4fc90umuj4.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.eyvn7flhlb4xtvo.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.gon14xphovlcp6p.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.hwuutl086asa7x2.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.i1ypljgc5u9gulj.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.lkwxa5jxocp5cdw.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.mb501chi8fnqgmm.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.mz7vgmcf23rofcc.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.nejmgf7sart3hfv.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.pqmjx25qpxkduz7.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.rv1tsn3lz95y1o0.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.v6ozwtpojmqfurc.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.vmjdi902sqi9ih3.rcgu.o" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.wcd421u3yf9l9xk.rcgu.o" "-o" "/Users/rsimmons/dev/rustler/target/debug/deps/librustler_test.dylib" "-Wl,-exported_symbols_list,/var/folders/2s/_21w072d06n7bx9d1mpwy6zm0000gn/T/rustc.WGEnusd98CVc/list" "/Users/rsimmons/dev/rustler/target/debug/deps/rustler_test.crate.allocator.rcgu.o" "-Wl,-dead_strip" "-nodefaultlibs" "-L" "/Users/rsimmons/dev/rustler/target/debug/deps" "-L" "/Users/rsimmons/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib" "/Users/rsimmons/dev/rustler/target/debug/deps/librustler-11859f1af9338cc0.rlib" "/Users/rsimmons/dev/rustler/target/debug/deps/liblazy_static-06ef2c3f75526230.rlib" "/Users/rsimmons/dev/rustler/target/debug/deps/liberlang_nif_sys-63257abef9730268.rlib" "/Users/rsimmons/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libstd-47f9a7ab54e7fd55.rlib" "/Users/rsimmons/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libpanic_unwind-34945e660c1550db.rlib" "/Users/rsimmons/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libunwind-2582c6ee1d80b6ff.rlib" "/Users/rsimmons/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/liballoc_system-7c3c30d2aa65bfa1.rlib" "/Users/rsimmons/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/liblibc-721ecb302a549dca.rlib" "/Users/rsimmons/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/liballoc-32f8204d813d7d31.rlib" "/Users/rsimmons/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libstd_unicode-771b444d7a99c1b3.rlib" "/Users/rsimmons/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libcore-0c8375020799bd23.rlib" "/Users/rsimmons/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/x86_64-apple-darwin/lib/libcompiler_builtins-95d524c979807816.rlib" "-l" "System" "-l" "resolv" "-l" "pthread" "-l" "c" "-l" "m" "-dynamiclib" "-Wl,-dylib"
  = note: Undefined symbols for architecture x86_64:
            "_enif_snprintf", referenced from:
                rustler::wrapper::term::fmt::hd9de1f4455b98448 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.t3wc3ngapyvghfi.rcgu.o)
            "_enif_get_local_pid", referenced from:
                rustler::wrapper::pid::get_local_pid::hcac7580812b2c73a in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.2ylrdwbbpcdaezf5.rcgu.o)
            "_enif_map_iterator_get_pair", referenced from:
                rustler::wrapper::map::map_iterator_get_pair::h6d9b72fe71d12961 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.3le968n251d32616.rcgu.o)
            "_enif_map_iterator_destroy", referenced from:
                rustler::wrapper::map::map_iterator_destroy::hbf24ca1d7cc4a04c in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.3le968n251d32616.rcgu.o)
            "_enif_map_iterator_create", referenced from:
                rustler::wrapper::map::map_iterator_create::h5f651890107b579c in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.3le968n251d32616.rcgu.o)
            "_enif_make_ulong", referenced from:
                erlang_nif_sys::erlang_nif_sys_api::enif_make_uint64::h2cbcb4675d60aefa in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.12t83mjc4eegvzuq.rcgu.o)
            "_enif_inspect_binary", referenced from:
                rustler::wrapper::nif_interface::enif_inspect_binary::h02c63e516ae96c27 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_make_long", referenced from:
                erlang_nif_sys::erlang_nif_sys_api::enif_make_int64::h6c8fac1105a331ac in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.12t83mjc4eegvzuq.rcgu.o)
            "_enif_get_atom_length", referenced from:
                rustler::wrapper::nif_interface::enif_get_atom_length_latin1::h6ddd1e02454bc0f4 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_get_ulong", referenced from:
                erlang_nif_sys::erlang_nif_sys_api::enif_get_uint64::h07d9d4027ee5dc7b in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.12t83mjc4eegvzuq.rcgu.o)
            "_enif_get_long", referenced from:
                erlang_nif_sys::erlang_nif_sys_api::enif_get_int64::h0a049285337d67d5 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.12t83mjc4eegvzuq.rcgu.o)
            "_enif_get_map_value", referenced from:
                rustler::wrapper::nif_interface::enif_get_map_value::h498e18f116facfcd in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_send", referenced from:
                rustler::wrapper::nif_interface::enif_send::hae8c732992b75392 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_keep_resource", referenced from:
                rustler::wrapper::nif_interface::enif_keep_resource::h61f26b2b483d060a in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_release_resource", referenced from:
                rustler::wrapper::nif_interface::enif_release_resource::hf414a7d61e80090b in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_make_resource", referenced from:
                rustler::wrapper::nif_interface::enif_make_resource::hc0db35cf219f0ee4 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_make_reverse_list", referenced from:
                rustler::wrapper::nif_interface::enif_make_reverse_list::h3f477e1793c344fe in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_make_uint", referenced from:
                rustler::types::primitive::_$LT$impl$u20$rustler..types..Encoder$u20$for$u20$u32$GT$::encode::he14c965548d2ae7d in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.2whv8nfaa5dbe3kz.rcgu.o)
                rustler::types::primitive::_$LT$impl$u20$rustler..types..Encoder$u20$for$u20$u8$GT$::encode::h5f0f1671284c0067 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.2whv8nfaa5dbe3kz.rcgu.o)
            "_enif_release_binary", referenced from:
                rustler::wrapper::nif_interface::enif_release_binary::hfce7facdf249d0c2 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_alloc_resource", referenced from:
                rustler::wrapper::nif_interface::enif_alloc_resource::h7eb852d38457d507 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_thread_type", referenced from:
                rustler::wrapper::nif_interface::enif_thread_type::h25a0a0f291050088 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_make_int", referenced from:
                rustler::types::primitive::_$LT$impl$u20$rustler..types..Encoder$u20$for$u20$i32$GT$::encode::he9476de4141a812e in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.2whv8nfaa5dbe3kz.rcgu.o)
            "_enif_make_list_from_array", referenced from:
                rustler::wrapper::nif_interface::enif_make_list_from_array::hec89e310d4dfe2f7 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_make_tuple_from_array", referenced from:
                rustler::wrapper::nif_interface::enif_make_tuple_from_array::ha49238bf2f128d65 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_open_resource_type", referenced from:
                rustler::wrapper::nif_interface::enif_open_resource_type::h19f271d6cafbc041 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_make_new_map", referenced from:
                rustler::wrapper::nif_interface::enif_make_new_map::h53858f2b880aa325 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_make_map_put", referenced from:
                rustler::wrapper::nif_interface::enif_make_map_put::h2ec2f2b444be67ad in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_make_binary", referenced from:
                rustler::wrapper::nif_interface::enif_make_binary::h7ec37aaab780736b in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_realloc_binary", referenced from:
                rustler::wrapper::nif_interface::enif_realloc_binary::hc1fd24a49cb1e7be in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_is_empty_list", referenced from:
                rustler::wrapper::nif_interface::enif_is_empty_list::h58837a1d84450de9 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_get_atom", referenced from:
                rustler::wrapper::nif_interface::enif_get_atom_latin1::hfa61e13ab63fd93d in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_is_identical", referenced from:
                rustler::wrapper::nif_interface::enif_is_identical::he890c36f62535252 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_get_list_cell", referenced from:
                rustler::wrapper::nif_interface::enif_get_list_cell::h02b37bd54ab26507 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_make_sub_binary", referenced from:
                rustler::wrapper::nif_interface::enif_make_sub_binary::h0e403fa099bcc9bb in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_get_uint", referenced from:
                rustler::types::primitive::_$LT$impl$u20$rustler..types..Decoder$LT$$u27$a$GT$$u20$for$u20$u32$GT$::decode::habc330846e8a6748 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.2whv8nfaa5dbe3kz.rcgu.o)
                rustler::types::primitive::_$LT$impl$u20$rustler..types..Decoder$LT$$u27$a$GT$$u20$for$u20$u8$GT$::decode::hc9a05dc804892688 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.2whv8nfaa5dbe3kz.rcgu.o)
            "_enif_is_atom", referenced from:
                rustler::wrapper::nif_interface::enif_is_atom::ha5e95b4fb7f5aa4a in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_make_copy", referenced from:
                rustler::wrapper::nif_interface::enif_make_copy::h481846b797b1c7f3 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_is_list", referenced from:
                rustler::wrapper::nif_interface::enif_is_list::hb1ede9b3e8bca14f in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_make_list_cell", referenced from:
                rustler::wrapper::nif_interface::enif_make_list_cell::hafe5abe53b63b6a4 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_get_double", referenced from:
                rustler::types::primitive::_$LT$impl$u20$rustler..types..Decoder$LT$$u27$a$GT$$u20$for$u20$f64$GT$::decode::hf14952e6fd559588 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.2whv8nfaa5dbe3kz.rcgu.o)
            "_enif_raise_exception", referenced from:
                rustler::wrapper::nif_interface::enif_raise_exception::haeebcd2fb5ba4d54 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_get_tuple", referenced from:
                rustler::wrapper::nif_interface::enif_get_tuple::he97792aa26a0efc3 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_make_badarg", referenced from:
                rustler::wrapper::nif_interface::enif_make_badarg::hfa5b31057b7938c7 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_alloc_binary", referenced from:
                rustler::wrapper::nif_interface::enif_alloc_binary::h6a04dfe902713cd7 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_make_double", referenced from:
                rustler::types::primitive::_$LT$impl$u20$rustler..types..Encoder$u20$for$u20$f64$GT$::encode::hc78f7a38e4168c2a in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.2whv8nfaa5dbe3kz.rcgu.o)
            "_enif_free_env", referenced from:
                _$LT$rustler..env..OwnedEnv$u20$as$u20$core..ops..drop..Drop$GT$::drop::h27f7be652b5eabff in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.7ve22dj4c6tgj5o.rcgu.o)
            "_enif_get_resource", referenced from:
                rustler::wrapper::nif_interface::enif_get_resource::h03be5cda294a98aa in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_map_iterator_next", referenced from:
                rustler::wrapper::map::map_iterator_next::h9ae785eb23900019 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.3le968n251d32616.rcgu.o)
            "_enif_make_atom_len", referenced from:
                rustler::wrapper::nif_interface::enif_make_atom_len::h0417e18b209a3da7 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_compare", referenced from:
                rustler::wrapper::nif_interface::enif_compare::he746541a04a4610e in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_get_int", referenced from:
                rustler::types::primitive::_$LT$impl$u20$rustler..types..Decoder$LT$$u27$a$GT$$u20$for$u20$i32$GT$::decode::hfab4831f49031cd7 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.2whv8nfaa5dbe3kz.rcgu.o)
            "_enif_alloc_env", referenced from:
                rustler::wrapper::nif_interface::enif_alloc_env::h531bee338e720370 in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.1oag4ktxo6t0wz1b.rcgu.o)
            "_enif_self", referenced from:
                rustler::types::pid::_$LT$impl$u20$rustler..env..Env$LT$$u27$a$GT$$GT$::pid::h8bbc50419a9ec36b in librustler-11859f1af9338cc0.rlib(rustler-11859f1af9338cc0.7ve22dj4c6tgj5o.rcgu.o)
          ld: symbol(s) not found for architecture x86_64
          clang: error: linker command failed with exit code 1 (use -v to see invocation)
          

error: aborting due to previous error

error: Could not compile `rustler_test`.

To learn more, run the command again with --verbose.
12:56:43 ➜  ~/dev/rustler ➜  git:(master) 
```

--- a/.cargo/config
+++ b/.cargo/config
@@ -1,0 +1,5 @@
+[target.x86_64-apple-darwin]
+rustflags = [
+    "-C", "link-arg=-undefined",
+    "-C", "link-arg=dynamic_lookup",
+]

